protobuf/source_reflection.go

```diff
@@ -95,7 +98,7 @@ func (s *grpcreflectSource) FindService(fullyQualifiedName string) (*desc.Servic
 		}
 
 		if !grpcreflect.IsElementNotFoundError(err) {
-			return nil, err
+			return nil, fmt.Errorf("error in protobuf reflection: %v", err)
```

Comment:
Couple of things to check:
 * Should we add this same wrapper for all errors that are unexpected (e.g., in `FindMessage` as well)
 * slightly related to this change, but it's an existing issue: should we check `IsElementNotFound` before we even do `ListServices`?

Reply:
Wrapped the errors in `FindMessage` and `ListServices` with the same message. Ordering of `ListServices` was above to avoid checking `IsElementNotFound` twice, I have reordered them with an additional `IsElementNotFound` check for `ListServices`